examples/node-app/src/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Welcome to Cloudflare Workers! This is your first worker.
+ *
+ * - Run `wrangler dev` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your worker in action
+ * - Run `wrangler publish --name node-app` to publish your worker
+ *
+ * Learn more at https://developers.cloudflare.com/workers/
+ */
+
+import * as path from "path";
+import Stripe from "stripe";
+
+// make sure Stripe imports fine
+console.log(Stripe);
+
+export default {
+  async fetch() {
+    // make sure path actually works
+    return new Response(path.join("path/to", "some-file"));
```

Comment:
just a string, 'path/to/some-file'. This project is simply a sanity test if and when you need to test node-compat. Honestly I might just delete this folder since it's not like we run it during CI or anything.